pkg/operator/operator.go

```diff
@@ -113,7 +113,22 @@ type Operator struct {
 func NewOperator() (context.Context, *Operator) {
 	// Root Context
 	ctx := signals.NewContext()
-	ctx = knativeinjection.WithNamespaceScope(ctx, system.Namespace())
+	sysnsKey := "SYSTEM_NAMESPACE"
+	systemNamespace := env.WithDefaultString(sysnsKey, "")
```

Comment:
This was my first thought, but adding it to option means flag support is needed, right? I'd be happy to add it to option if  that's right side.

Reply:
Actually looking at this deeper -- can we just drop this environment variable outright? Once we drop webhook support, I don't think we even need this environment variable because the leader election config will default to the pod namespace by default anyways AND leader election should just be disabled when running locally IMO

Reply:
absolutely, since v1.1 will drop  this webhook support, so no need to add more complexity here.

I delete the env SYSTEM_NAMESPACE.